infra.bs

```diff
@@ -628,38 +628,40 @@ them effectively as-is.
 of the <a>code units</a> 0xD83D, 0xDCA9, and 0xD800, when interpreted as containing
 <a>code points</a>, would consist of the <a>code points</a> U+1F4A9 and U+D800.
 
-<p>A <dfn export>scalar value string</dfn> is a sequence of <a>scalar values</a>.
+<p>A <a>string</a>'s <dfn export for=string>code unit length</dfn> is the number of
+<a>code units</a> it contains. For now, this can also be referred to as a <a>JavaScript string</a>'s
+<dfn export for="JavaScript string">length</dfn>.
+
+<p>A <a>string</a>'s <dfn export for=string>code point length</dfn> is the number of
+<a>code points</a> it contains. For now, this can also be referred to as a <a>string</a>'s
+<dfn export for=string>length</dfn>.
```

Comment:
Oof, now that this is explicit, it is really confusing. Can we pick only one that is the string's "length", and delete the other? And hope that doesn't break things in the ecosystem too badly?

I think "code unit length" is most likely to be correct since it matches JavaScript thus IDL...

Reply:
Yeah, discussing  this with @andreubotella I think we want "length" and "code point length" and probably use the former if it doesn't matter or is better?

The main problem is redefining "length" for string as currently that means code point length. Maybe that's okay though and this hasn't seen much adoption yet. @andreubotella was looking into that as well.